docs/ha.md

```diff
@@ -130,4 +130,4 @@ Immediatly after fencing, **Minion 1** will be booted on the other host.
 
 #### Pull the network cable
 
-Finally, the worst case: leaving the storage operational but "cut" the (management) network interface. Same procedure: unplug physically the cable, and wait... Because **lab1** can't contact any other host of the pool (in this case, **lab2**), it decides to start the fencing procedure. The result is exaclty the same as the previous test. It's gone for the pool master, displayed as "Halted" until we re-plug the cable.
```

Comment:
github does not show a diff here, this second change is adding a missing newline at EOF, good :)